src/doc/using_visit/WorkingWithFiles/Supported_File_Types.rst

```diff
@@ -42,7 +42,9 @@ More than one ``-fallback_format`` option can be specified on the command-line a
 
 Finally, you can also specify the plugin to use to open a file as part of the ``-o`` command-line option.
 For example, to open the file ``foobar.gorfo`` as a Silo file, you can specify ``-o foobar.gorfo,Silo_1.0`` on the command-line when starting VisIt_.
-Note this feature of the ``-o`` option *requires* the plugin name (in correct case) followed by an underscore (``_``) and then its version number which is almost always ``1.0``.
+Note this feature of the ``-o`` option *requires* the plugin name (in correct case) followed by an underscore (``_``) and then its version number which is almost always ``1.0``. 
```

Comment:
So it turns out this was in fact documented. I added to what was here for more clarity.